packages/postgres-database/src/simcore_postgres_database/utils.py

```diff
@@ -25,7 +25,7 @@ def build_url(
         user=user,
         password=password,
         host=host,
-        port=port,
+        port=int(port),
```

Comment:
not sure I get why you need to cast here. the port is defined as an int above???

Reply:
haha this one drove me crazy! It is a modification I did in a bigger PR https://github.com/ITISFoundation/osparc-simcore/pull/5346 and moved it here to prevent the same situation if somebody updates key libraries.
What happened there is that some library (still not sure which) changed the type in which that port was passed or `yarl` does not accepts anymore `str`. This little thing creates a mess in the tests since it is used to connect to the database and many many tests fail.